tests/runner/src/main/resources/META-INF/mule-module.properties

```diff
@@ -0,0 +1,23 @@
+module.name=runner
+
+artifact.export.classPackages=org.eclipse.aether.resolution,\
+                              org.eclipse.aether.version,\
+                              org.eclipse.aether.artifact,\
+                              org.eclipse.aether.transfer,\
+                              org.eclipse.aether,\
+                              org.eclipse.aether.installation,\
+                              org.eclipse.aether.graph,\
+                              org.eclipse.aether.collection,\
+                              org.eclipse.aether.deployment,\
+                              org.eclipse.aether.repository,\
+                              org.eclipse.aether.metadata,\
+                              org.mule.test.runner,\
+                              org.junit.runners.model,\
+                              org.junit.runner.notification,\
+                              org.junit.runner,\
+                              junit.framework,\
+                              org.junit.runner.manipulation
+
+
+artifact.export.resources=/META-INF,\
+                          /META-INF/services/org/mule/runtime/core/config
```

Comment:
I moved the packages that were included in the module properties from the functional test module. I assume that they were required